### PR TITLE
feat(whitepaper): translate to spanish

### PR DESCRIPTION
**Pull request by developerfred**
Ref: https://github.com/BarnBridge/Bounty-Program/issues/3

- [x] Whitepaper  ✅ 
- [x]  Website ->  [Translate Spanish doc](https://docs.google.com/document/d/1rztk1tVEb1KiylDPErcm4_HIUyuRXzA0Qm757WABz8A/edit?usp=sharing)

### CODE REVIEW

**Comment by kkkroz** [comment]
I can see it is about 2,000 words. Do you still need the review? Sorry about the delay, but I never used this platform before and I am still trying to understand how it works.

**Comment by developerfred** [comment]
@kkkroz https://github.com/BarnBridge/BarnBridge-Whitepaper/pull/12/files, There is a green button to view or review, so you can open and write your considerations.

**Comment by kkkroz** [comment]
So, you don't need a proofreading. Do you just need to check the overall quality ?
Sorry, but the task is not clear to me.

**Comment by kkkroz** [comment]
Or, do you need the proofreading?

**Comment by developerfred** [comment]
@kkkroz In this bounty you need to analyze the text to see if there is any spelling error, verbal agreement. If the protocol message is easily understood in Spanish.

**Comment by kkkroz** [comment]
OK. sure. But do I have to correct the potential errors?

**Comment by kkkroz** [comment]
I need also the original in English to have a perfect understanding of the translation. And et me know the deadline. Best

**Comment by developerfred** [comment]
@kkkroz You point out the errors, I update the pull request if necessary.
-> [Original in English](https://github.com/BarnBridge/BarnBridge-Whitepaper/blob/master/README.md)

**Comment by kkkroz** [comment]
Ok. I guess I understood. It is a complete different way to approach the correction from the one I used to. Wouldn't be quicker to just correct the errors?

**Comment by developerfred** [comment]
@kkkroz yes, you can place your corrections here on this page, according to the image. [review changes]
Screen Shot 2020-12-15 at 11 17 20

**Comment by kkkroz** [comment]
ok. Deadline?

**Comment by kkkroz** [comment]
Sorry I can't help on this task. Can't understand how this platform work. My apologies.

File: README.es.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+ 
+La necesidad de que existan instrumentos TradFi conocidos en todo el ecosistema DeFi nunca ha sido más fuerte. **BarnBridge es una idea cuyo tiempo ha llegado.**
```
Comment (aadorian) [inline README.es.md:14]: Sugeriría **"BarnBridge es una idea cuyo momento ha llegado.**

File: README.es.md [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
... de riesgo de poseer, prestar o recibir un activo digital completo y (2) nunca se tomarán el tiempo para usar un  organización autónoma descentralizada (DAO) para crear un contrato inteligente que codifica algorítmicamente ambos lados del préstamo o acuerdo. Más del 99,9% de la deuda mundial todavía está estructurada a través de los mercados tradicionales y está hambrienta de rendimiento. Por el contrario, las empresas financieras más avanzadas no tienen tolerancia al riesgo. Esto permite diferentes estructuras en cada punto de la curva de rendimiento con los más riesgosos (probablemente fondos de cobertura) que desean depositar la menor cantidad de dinero con el mayor rendimiento para su apuesta/cobertura. Por el contrario, los inversores más conservadores suelen estar dispuestos a renunciar a una gran parte de las oportunidades alcistas para acceder a instrumentos más seguros. Los productos “sin riesgo”, como los describe tradFi, no se ofrecen actualmente en el ecosistema financiero descentralizado. La oportunidad de estructurar este tipo de instrumentos permitirá que los inversores más reacios al riesgo en los mercados tradicionales se muevan hacia los mercados descentralizados.
```
Comment (aadorian) [inline README.es.md:28]: sugeriría cambiar morder la curva de riesgo por atravesar la curva de riesgo

File: README.es.md [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+
+Planeamos crear el primer protocolo de derivados multiplataforma para todas y cada una de las fluctuaciones. Para empezar, nos centraremos en la sensibilidad al rendimiento y el precio de mercado. En sentido descendente, planeamos introducir una variedad mucho más amplia de coberturas contra las fluctuaciones en el ecosistema descentralizado. BarnBridge tiene como objetivo ser independiente de la plataforma y los activos.
+
+Puede reducir el riesgo de los activos digitales y la sensibilidad al rendimiento de los activos digitales dividiéndolos en fragmentos o tramos esencialmente infinitos, separados y denominados en dólares, y creando derivados a partir de estos tramos. BarnBridge tiene como objetivo suavizar la curva de riesgo y ofrecer una gestión de riesgo por capas a los inversores DeFi y tradFi mediante la creación de derivados basados ​​en deuda y rendimiento más eficientes.
```
Comment (aadorian) [inline README.es.md:41]: En este sentido sería posible...

File: README.es.md [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+
+![](images/image10.png)
+
+#### 2.4 UI/UX Interface (Dark) 
+
+![](images/image13.png)
+
+## 3. Token - $BOND
+
+BOND es un token ERC-20. Se utilizará para participar en el sistema y como símbolo de gobernanza cuando se lance el módulo de gobernanza.  Como cumple con el estándar ERC-20, el token $BOND se puede intercambiar en cualquier intercambio y se puede almacenar en cualquier billetera, lo que permite que cualquier persona en el mundo acceda a él.
```
Comment (aadorian) [inline README.es.md:101]: en lugar de utilizar intercambio, sugeriría utilizar jargon utilizado por el sistema crypto directamente como exchange ..

File: README.es.md [hunk 5]
```diff
@@ -0,0 +1,262 @@
...
+
+### 6.2 Asesores/Socios tecnológicos
+
+- **Aaron McDonald** - Cofundador y director ejecutivo Aaron es un veterano de la industria tecnológica de 20 años con experiencia en la dirección de equipos en todos los aspectos de una empresa de tecnología que gestiona carteras de más de mil millones de dólares. Aaron fundó Centrality, un estudio de riesgo global líder de $ 100 millones que respalda una cartera de riesgo que aprovecha la tecnología descentralizada para crear nuevas innovaciones de mercado y experiencias de clientes. Aaron es miembro de la junta o asesor de más de una docena de empresas de riesgo en todo el mundo. En 2018, Aaron fue galardonado con el premio EY Entrepreneur of the Year en la categoría de tecnología e industrias emergentes.
```
Comment (aadorian) [inline README.es.md:251]: sugeriría cambiar veterano por experto

Comment (developerfred) [reply README.es.md]: thanks on #3117ecc

File: README.es.md (hunk 4 above)
Comment (developerfred) [reply README.es.md]: done on 3117ecc

**Pull request merged**